docs/pages/versions/unversioned/sdk/in-app-purchases.md

```diff
@@ -0,0 +1,304 @@
+
+#### Returns
+
+- **responseCode (_ResponseCode_)** -- A number indicating the last billing response code.
+
+#### Example
+```javascript
+const responseCode = await getBillingResponseCodeAsync();
+if (responseCode !== ResponseCode.OK) {
+    // Either we're not connected or the last response returned an error (Android)
+}
+```
+
+### `InAppPurchases.disconnectAsync()`
+
+Removes listeners, both internally and the one set in `setPurchaseListener`, and cleans up memory. Call this when you are done using the In-App Purchases API in your app.
+
+No other methods can be used until the next time you call `connectAsync`.
+
+#### Example
+```javascript
+await disconnectAsync();
+```
+
+## Object Types
+
+### QueryResponse
+
+The response type for queries and purchases.
+
+| Field name            | Type      | Platforms | Description                                                                   | Possible values                                                                                                                                                                                                                                                                                                                                                    |
+| --------------------- | --------- | --------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| responseCode                 | _ResponseCode_  | both      | The response code from a query or purchase.                                  | `ResponseCode.OK`, `ResponseCode.USER_CANCELED`, `ResponseCode.ERROR`, `ResponseCode.DEFERRED` (iOS only)                                                                                                                                                                                                                                                                                                                                                                   |
+| results   | _Purchase[] or ItemDetails[]_ | Android      | The array containing the `Purchase` or `ItemDetails` objects requested depending on the method.           |                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I thought results was both platforms, but for iOS it would just contain one item?

Reply:
No, both return an array for consistency but iOS will typically only have one entry in it, at least when you're making a purchase.

Reply:
Right, but it lists `Android` under platforms instead of `both`- since both return an array (even though iOS is an array of 1), shouldn't it say `both` under Platforms?

Reply:
Oh whoops, that was totally a mistake. Good catch 👍